pkg/consts/consts.go

```diff
@@ -1,10 +1,14 @@
 package consts
 
 const (
-	ActionsetNameKey         = "ActionSet"
-	PodNameKey               = "Pod"
-	ContainerNameKey         = "Container"
-	PhaseNameKey             = "Phase"
+	ActionsetNameKey = "ActionSet"
+	PodNameKey       = "Pod"
+	ContainerNameKey = "Container"
+	PhaseNameKey     = "Phase"
+	LogKindKey       = "LogKind"
+	LogKindSystem    = "system"
```

Comment:
I think we are not u sing `LogKindSystem` anywhere.

Reply:
Good catch - thanks.